app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -486,8 +502,7 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
             b.settingsActivityThemeRl.isEnabled = false
             showDialogForTheme()
             object : CountDownTimer(500, 500) {
-                override fun onTick(millisUntilFinished: Long) {
-                }
+                override fun onTick(millisUntilFinished: Long) {}
```

Comment:
see below.

Reply:
he countDownTimer is to disable the UI element for 500ms. And will be enabled after that.
isAdded is the component of Fragment.
Either we can use Handler or CountDownTimer so chose to use CountDownTimer instead of Handler.